CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 3.2.0
+* CHANGE: Callbacks set via `setDataCallback`, `setShouldSendCallback` now receive any prior-set callback as the 2nd argument. See: https://github.com/getsentry/raven-js/pull/636
+* CHANGE: Raven.js no longer passes a 'message' interface for exceptions. See: https://github.com/getsentry/raven-js/pull/632
```

Comment:
Oh, I didn't even realize that was a server change. Hmm, what's the right move here given that this could be breaking for on-premise users. Do we go to a full version bump (4.0.0)?

cc @dcramer